demos/common/performance_metrics/metrics.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <chrono>
+
+#include "../samples/ocv_common.hpp"
+
+class PerformanceMetrics {
+public:
+    using Clock = std::chrono::steady_clock;
+    using TimePoint = std::chrono::time_point<Clock>;
+    using Duration = Clock::duration;
+    using Us = std::chrono::microseconds;
+    using Ms = std::chrono::milliseconds;
```

Comment:
While `Duration` should be represented as `Clock::duration`, it is better to represent `Ms` and `Sec` as `std::chrono::duration<double, ...>` because they are just helpers that simplify result conversion.